books/RayTracingTheRestOfYourLife.html

```diff
@@ -312,30 +312,143 @@
+
+    $$ y = f(x) $$
+    $$ Average(y) \Big|_{a \leq x \leq b} \approx Average(y_i) \Big|_{a \leq x_i \leq b} $$
+
+If we take the number of samples $N$ and take the limit as $N$ goes to $\infty$, then we get the following:
+
+    $$ Average(y) \Big|_{a \leq x \leq b} \approx \frac{1}{N} \sum_{i=0}^{N-1} f(x_i)
+        \Big|_{a \leq x_i \leq b} $$
+    $$ Average(y) \Big|_{a \leq x \leq b} = \lim_{N \to +\infty} \frac{1}{N} \sum_{i=0}^{N-1} f(x_i)
+        \Big|_{a \leq x_i \leq b} $$
+
+The average value of $f(X)$ over the continuous interval $[a, b]$ can be perfectly represented by
+summing over an infinite number of random points within that interval. As this number of points
+approaches $\infty$ the average of the outputs approaches the exact answer. This is a MC algorithm.
 
+We don't need to take random points to solve for the exact answer over the interval. We also have
+the option to place the sampling points at distinct locations. If we had $N$ samples over an
+interval $[a,b]$ then we could choose equally spaced points throughout:
+
+    $$ x_i = a + i \Delta x $$
+    $$ \Delta x = \frac{b - a}{N} $$
+
+Then solving for their average:
+
+    $$ Average(y) \Big|_{a \leq x \leq b} \approx \frac{1}{N} \sum_{i=0}^{N-1} f(x_i)
+        \Big|_{x_i = a + i \Delta x} $$
+    $$ Average(y) \Big|_{a \leq x \leq b} \approx \frac{\Delta x}{b - a} \sum_{i=0}^{N-1} f(x_i)
+        \Big|_{x_i = a + i \Delta x} $$
+    $$ Average(y) \Big|_{a \leq x \leq b} \approx \frac{1}{b - a} \sum_{i=0}^{N-1} f(x_i) \Delta x
+        \Big|_{x_i = a + i \Delta x} $$
+
+Take the limit as $N$ approaches $\infty$
+
+    $$ Average(y) \Big|_{a \leq x \leq b} = \lim_{N \to +\infty} \frac{1}{b - a} \sum_{i=0}^{N-1}
+        f(x_i) \Delta x \Big|_{x_i = a + \Delta x * i} $$
+
+This is just a regular integral:
+
+    $$ Average(y) \Big|_{a \leq x \leq b} = \frac{1}{b - a} \int_{a}^{b}
+        f(x) dx $$
+    $$ \bar f = \frac{1}{b - a} \int_{a}^{b} f(x) dx $$
+
+Both the integral of a function and a MC over that function can be used to solve for the average
+over a specific interval. While integration solves for the sum of infinitesmally small slices of the
+interval, a MC will solve for the sum of infinitely many random points within that interval. The
+average over an interval is intrinsically linked with the area under the curve in that interval.
+
+    $$ \bar f = \frac{1}{b - a} \cdot area(f(x), a, b) $$
+
+Counting the number of points that fall inside of an object isn't the only way to measure it's
+average or area. Integration is also a common mathematical tool for this purpose. If a closed form
+exists for a problem, integration is frequently the most natural and clean way to formulate things.
+
+I think a couple of examples will help.
 
 Integrating x²
 ---------------
+<div class='together'>
 Let’s look at a classic integral:
 
-$$ I = \int_{0}^{2} x^2 dx $$
+    $$ I = \int_{0}^{2} x^2 dx $$
```

Comment:
Does this work? I think you have to indent two spaces; four spaces indicates pre-formatted text.